CHANGELOG.md

```diff
@@ -23,6 +23,24 @@ Dropping a requirement of a major version of a dependency is a new contract.
 ## [Unreleased]
 [Unreleased]: https://github.com/atlassian/aws-infrastructure/compare/release-2.23.1...master
 
+### Fixed
+- Restrict access to provisioned AWS instances to only the necessary connections. Resolve [JPERF-730].
+
+### Added
+- Add the ability to dynamically share permissions between resources provisioned by various Formulas. Unblock [JPERF-730].
+- Add builders for `ProvisionedJira`, `ProvisionedLoadBalancer`, `ProvisionedVirtualUsers` and `ProvisionedInfrastructure`.
+- Add `Defaults` child object to `ProvisionedJira`, `ProvisionedLoadBalancer`, `ProvisionedVirtualUsers`, `ProvisionedInfrastructure`, `StandaloneFormula` and `DataCenterFormula`.
```

Comment:
@dagguh this is application of your suggestion. Is this CHANGELOG entry about `Defaults` good enough or would you suggest framing it differently?

Reply:
Looks good